src/bci_build/package/cosign/README.md.j2

```diff
@@ -0,0 +1,57 @@
+# {{ image.title }}
+{% include 'badges.j2' %}
+
+## Description
+Cosign aims to make signatures invisible infrastructure.
```

Comment:
I don't understand what this means. What's "invisible infrastructure"?

Reply:
Then perhaps something like _Cosign is designed to make managing signatures easy and transparent_?

Reply:
I don't want to change how upstream wants to position the project. anyhow, I updated it a bit, I hope it is fine now.